git.py

```diff
@@ -52,7 +52,7 @@ def get_latest_tag_commit(self, pattern):
 
     def update_repository(self):
         self._git_redirected_success("checkout '{}'".format(self.branch))
-        self._git_redirected_success("pull origin '{}'".format(self.branch))
+        self._git_redirected_success("pull --tags origin '{}'".format(self.branch))
```

Comment:
Was reading http://stackoverflow.com/questions/1204190/does-git-fetch-tags-include-git-fetch and it had some interesting things to say about combining pull and --tags as well as when fetch --tags included fetch without tags.  We are probably on a new enough git version for it to work, but would it be cleaner and safer to do two commands, a fetch --tags and the current git pull?